src/V3AstNodeOther.h

```diff
@@ -1248,6 +1248,8 @@ class AstNetlist final : public AstNode {
     AstCFunc* m_evalp = nullptr;  // The '_eval' function
     AstCFunc* m_evalNbap = nullptr;  // The '_eval__nba' function
     AstVarScope* m_dpiExportTriggerp = nullptr;  // The DPI export trigger variable
+    using IfaceTriggerVec = std::vector<std::pair<const AstIface*, AstVarScope*>>;
+    IfaceTriggerVec m_virtIfaceTriggerps;  // The virtual interface trigger variables
```

Comment:
Can we somehow recompute this when needed based on other tree contents?  Keeping nodes in structures other than direct tree children is a maintenance mess I'd generally prefer to avoid - most things stored here in netlist are items that once made never go away.  As a direct example you forgot to implement the clone() repair methods.

Reply:
I could try storing the trigger vars as children of the interfaces they correspond to.

Reply:
Yes, as children seems good.  I'm ok with a sub-stage too, but find it much easier to understand and debug 2 smaller steps with a tree in the middle versus a bigger one.

Reply:
Moved it to a `V3Sched` sub-stage. I think it makes sense, as this change is all about scheduling. You can use `--dumpi-tree 6` to get a dump after `V3SchedVirtIface`, as with other `V3Sched` sub-stages.